src/test/lib/scheduler/operator_task_test.cpp

```diff
@@ -173,4 +173,30 @@ TEST_F(OperatorTaskTest, UncorrelatedSubqueries) {
   }
 }
 
+TEST_F(OperatorTaskTest, SkipOperatorTaskTest) {
+  const auto gt = std::make_shared<GetTable>("table_a");
+  gt->execute();
+
+  const auto task = std::make_shared<OperatorTask>(gt);
+  EXPECT_NO_THROW(task->skip_operator_task());
```

Comment:
why `EXPECT_NO_THROW` rather than just calling it?

Reply:
In other parts of the code there was also an insertion instead of just throwing the error, so we wanted to make it consistent

Reply:
True, not your fault. We really discussed the use of `ECPECT_NO_THROW` in this test file the last time we added a test here. Our final decision was that it is okay when the execution of a called method was mainly dependent of another component (the scheduler in this case). Since everything in this tested method stays in OperatorTask (or is mainly some dummy execution), it should not be wrapped whithin the `EXPECT_NO_THROW` makro (since we could basically wrapp all happy-path code inside it).